cc/mcts_node.cc

```diff
@@ -304,17 +305,37 @@ void MctsNode::IncorporateResults(absl::Span<const float> move_probabilities,
         legal_moves[i] ? policy_scalar * move_probabilities[i] : 0;
 
     edges[i].original_P = edges[i].P = move_prob;
-    // Initialize child Q as current node's value, to prevent dynamics where
-    // if B is winning, then B will only ever explore 1 move, because the Q
-    // estimation will be so much larger than the 0 of the other moves.
+
+    // NOTE: Minigo uses value {-1, 1} from black's perspective
+    //       Leela uses value {0, 1} from current player's perspective (?)
+    //       AlphaGo uses ???
+    //       ELF uses ???
+    //
+    // The initial value of a child's Q is not perfectly understood.
     //
-    // Conversely, if W is winning, then B will explore all 362 moves before
-    // continuing to explore the most favorable move. This is a waste of
-    // search.
+    // There are a couple of general ideas:
+    //   * Init to Parent: Init a new child to it's parent value
+    //      We think of this as saying "The game is probably the same after *any* move
+    //   * Init to Draw (AKA init to zero): Init a new child to 0 AKA "position looks even":
```

Comment:
AKA "the game looks even on any unexplored subtree"

Reply:
not true because of policy. Let's get something in and we (READ: you) can clean up wording later if you still care